### ISSUE TITLE
Unable to save big PDF file

### ISSUE THREAD

**Issue by j0k3r** [wallabag/wallabag#3824]
While saving a big PDF file, the process is dying because it goes out of memory (on wallabag.it)
For example using http://sanjose.granicus.com/MetaViewer.php?meta_id=638815

Maybe we should try to find a way to avoid parsing big PDF and only save the title?

**Comment by Kdecherf** [comment]
We could use `pdftotext` when available to extract the text, thoughts?

**Comment by Kdecherf** [comment]
(extraction is instant on the linked pdf with this tool)

**Comment by j0k3r** [comment]
But it's an external binary, right?

**Comment by Kdecherf** [comment]
Yep, but I think it's not an issue

**Comment by j0k3r** [comment]
But we'll need user to install it. That might be the problem.
Maybe we can still use the actual solution and test if the binary exist to use it?

**Comment by Kdecherf** [comment]
Yes, it was implicit to me: we could use pdftotext only if available, otherwise we use tcpdf.
